<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoDanmarkCheckerDialogBase</class>
 <widget class="QDialog" name="GeoDanmarkCheckerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoDanmark Checker</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>720</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="RulesTreeWidget" name="rules_validate">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>105</y>
     <width>365</width>
     <height>300</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="before_dataset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Before dataset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="after_dataset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>After dataset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="before_dataset_input">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>23</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="after_dataset_input">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>63</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="RulesTreeWidget" name="rules_compare">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>410</y>
     <width>365</width>
     <height>300</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RulesTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>GeoDanmarkChecker.ui.rulestreewidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeoDanmarkCheckerDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeoDanmarkCheckerDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
